sdk/python/tests/integration/registration/test_universal_types.py

```diff
@@ -221,7 +221,12 @@ def test_feature_get_online_features_types_match(online_types_test_fixtures):
     features = [fv.name + ":value"]
     entity = driver(value_type=config.entity_type)
     fs.apply([fv, entity])
-    fs.materialize(environment.start_date, environment.end_date)
+    fs.materialize(
+        environment.start_date,
+        environment.end_date
+        - timedelta(hours=1)  # throwing out last record to make sure
```

Comment:
non blocking, but kind of confusing why this ensures we infer from all empty values